src/frontend/components/Button/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useLayoutEffect, useState, useRef } from 'react';
+import { Box, Button as GrommetButton, Text } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import ResizeObserver from 'resize-observer-polyfill';
+import styled from 'styled-components';
+
+import { SvgProps } from 'components/SVGIcons';
+import { theme } from 'utils/theme/theme';
+
+const StyledGrommetButton = styled(GrommetButton)`
+  height: 100%;
+  padding: 0;
+`;
+
+const StyledBox = styled(Box)`
+  display: flex;
+  flex: 1;
+  margin: 0 8px;
+  margin-bottom: 6px;
+  min-height: 0;
+  position: relative;
+  width: 100%;
+`;
+
+const StyledText = styled(Text)`
+  font-family: Roboto-Regular;
+  font-weight: normal;
+  letter-spacing: -0.13px;
+  white-space: nowrap;
+`;
+
+interface ButtonProps {
+  badge?: JSX.Element;
+  disabled?: boolean;
+  Icon?: React.FC<SvgProps>;
+  label?: string;
+  onClick?: () => void;
+  reversed?: boolean;
+  title?: string;
+}
+
+export const Button = ({
+  badge,
+  disabled,
+  Icon,
+  label,
+  onClick,
+  reversed,
+  title,
+}: ButtonProps) => {
+  const iconRef = useRef<HTMLDivElement | null>(null);
```

Comment:
```suggestion
  const iconRef = useRef<Nullable<HTMLDivElement>>(null);
```

We have a `Nullable` helper in `utils/types`

Reply:
https://github.com/openfun/marsha/pull/1249/commits/d17ee913fa6c32027eb189da6c0b124954f98c59